### PR TITLE
Free version of cookie-law-info breaks caching

### PR DESCRIPTION
**Pull request by rachelwhitton**
**Note:** Please fill out the PR Template to ensure proper processing.

Closes #

## Effect
PR includes the following changes:
- Add free GDPR Cookie Consent plugin to the WP doc for plugins with known issues, it breaks caching
- Include solution that upgrading to premium version and disabling script blocker resolves 

Note: I've not directly configured the premium version myself, but have confirmed this solution on Pantheon via an active migration client 

@alexfornuto can you put that fancy highlighting on the curl response to call out the two `set-header:` lines? Please :) 

## Remaining Work
- [ ] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5527 | Difference |
| --- | --- | --- | --- |
| Performance Score | `51` | `38` | `-13` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5527` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18542/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18542/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/plugins-known-issues.md [hunk 1]
```diff
@@ -338,6 +338,37 @@ ___
 
 ___
 
+## [GDPR Cookie Consent](https://wordpress.org/plugins/cookie-law-info/)
+
+<ReviewDate date="2020-02-20" />
```
Comment (rachelwhitton) [inline source/content/plugins-known-issues.md]: Might be a known issue already, but review date renders incorrectly - this says 20th, but rendered output produces 19th (not isolated to this instance, it appears widespread)

Comment (alexfornuto) [reply source/content/plugins-known-issues.md]: Yea, that's fixed in a pending commit: ce86eeda7a43c11c25c281c7fd76391c7c80238c

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5527 | Difference |
| --- | --- | --- | --- |
| Performance Score | `52` | `54` | `2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5527` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18554/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18554/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**